OrbitQt/main.cpp

```diff
@@ -54,6 +54,14 @@ ABSL_FLAG(bool, use_software_opengl, false, "Uses software implementation of "
 	      "OpenGL. Requires Mesa's opengl32.dll to be placed in the same "
 	      "folder as Orbit.");
 
+// TODO: Remove this flag once it can be specified in the ui.
```

Comment:
Maybe it is time to introduce CaptureOptions dialog?

Reply:
(create a bug and add link to it here)

Reply:
Linked new bug.